test/performance/performance.go

```diff
@@ -45,8 +50,11 @@ type Client struct {
 	PromClient *prometheus.PromProxy
 }
 
+// traceFile is the name of the
+var traceFile *os.File
+
 // Setup creates all the clients that we need to interact with in our tests
-func Setup(logf logging.FormatLogger, promReqd bool) (*Client, error) {
+func Setup(logf logging.FormatLogger, tName string, promReqd, tracing bool) (*Client, error) {
```

Comment:
Quoting one of the comments I received for in the discard PR https://github.com/knative/serving/pull/3436. Could you please use enumeration for the bool list of debugging options that can be enabled? 

> Signatures with multiple boolean arguments are nearly impossible to read.
> 
> One thing you can do in Go is define an enumeration to replace this:
> 
> ```
> type Direction int
> 
> const (
>     North Direction = iota
>     East
>     South
>     West
> )
> ```
> (source: https://yourbasic.org/golang/iota/)
> 
> This makes things much more readable:
> 
> Suggested change 
> ```
> 	perfClients, err := Setup(t.Logf, true, false)
> 	perfClients, err := Setup(t.Logf, EnableBacon, DisableSyrup)
> ```
> Commit suggestion 
> apparently I'm hungry :)

Reply:
sounds like a good option. Will do in a later PR.